lib/broccoli/default-packager.js

```diff
@@ -579,29 +579,14 @@ module.exports = class DefaultPackager {
         minifyCSS: this.minifyCSS.options,
       };
 
-      let stylesAndVendor, preprocessedStyles;
+      let stylesAndVendor = callAddonsPreprocessTreeHook(this.project, 'css', tree);
 
-      if (this._cachedProcessedSrc !== null && this.isModuleUnificationEnabled === true) {
-        stylesAndVendor = mergeTrees([
-          this._cachedProcessedSrc,
-          tree,
-        ], {
-          annotation: 'Module Unification Styles',
-        });
-        preprocessedStyles = new Funnel(this._cachedProcessedSrc, {
-          srcDir: `${this.name}/assets`,
-          destDir: 'assets',
-        });
-      } else {
-        stylesAndVendor = callAddonsPreprocessTreeHook(this.project, 'css', tree);
-
-        preprocessedStyles = preprocessCss(
-          stylesAndVendor,
-          '/app/styles',
-          '/assets',
-          options
-        );
-      }
+      let preprocessedStyles = preprocessCss(
+        stylesAndVendor,
+        this.isModuleUnificationEnabled === true ? 'src/ui/styles' : '/app/styles',
```

Comment:
Seems slightly inconsistent RE: the leading slash ( I'm not sure if it matters too much) and `someBoolean === true ? thing : other` seems redundant. What do you think of:

```suggestion
        this.isModuleUnificationEnabled ? '/src/ui/styles' : '/app/styles',
```

Reply:
https://github.com/ember-cli/ember-cli/pull/8289/commits/197aabe2dab76e2dbe9774692862a865118e3568